projects/showcase/test/mapping/virtual-keyboard.ts

```diff
@@ -35,39 +35,39 @@ export class VirtualKeyboard extends BasePage {
         return keys;
     }
 
-    public async clickKeys(text: string): Promise<void> {
+    public async tapOnKeys(text: string): Promise<void> {
         const keys: string[] = text.split('');
         for (let i = 0; i < keys.length; i++) {
-            await this.clickKey(keys[i]);
+            await this.tapOnKey(keys[i]);
         }
     }
 
-    public async clickBackspace(): Promise<void> {
-        await this.clickKey('{bksp}');
+    public async tapOnBackspace(): Promise<void> {
+        await this.tapOnKey('{bksp}');
```

Comment:
Better use the enum SystelabVirtualKeyboardButton to match keys. Same in all tapOnX methods

Reply:
Updated as suggested